src/signup.js

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import axios from "axios";
 import Constants from "./Constants";
 import Encryption from "./Encryption";
-import LoginNavBar from "./LoginNavBar";
-
+//change
```

Comment:
```suggestion

```

Reply:
this i put so that when i rename folder it shows due to some bug when i renamed folder it wasnt getting changed in git, I will COmmit this